packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -53,6 +53,7 @@ const trace = makeTracer('VM');
  * @param {ZCFMint<'nat'>} debtMint
  * @param {Brand} collateralBrand
  * @param {ERef<PriceAuthority>} priceAuthority
+ * @param {Ratio} debtPriceRatio not a tunable parameter; determined by priceAuthority
```

Comment:
`xPriceRatio` is redundant. A price is a ratio somewhat inherently. `debtPrice` is more consistent with most code I have read.

Reply:
As discussed, I'd like to open the bikeshedding for the appropriate name for this parameter.  My best suggestion so far is along the lines of `debtToPriceRatio`.

Basically, given a debt brand (`RUN`), and a price authority output brand (`USD`), have `debtToPriceRatio` of N RUN : M USD (N and M are different if the decimalPlaces don't match) so that debt amounts can be converted back and forth to price amounts.